webapp/channels/src/components/user_settings/notifications/user_settings_notifications.tsx

```diff
@@ -283,6 +287,9 @@ class NotificationsTab extends React.PureComponent<Props, State> {
             });
         }
         data.highlight_keys = highlightKeys.join(',');
+        if (this.props.user.notify_props?.highlight_keys !== data.highlight_keys) {
+            trackFeatureEvent(TrackPassiveKeywordsFeature, TrackPassiveKeywordsEvent);
```

Comment:
nit: Perhaps add the number of keywords added, removed and total as properties?

Reply:
This is not added as per discussed here https://docs.google.com/document/d/1u7UcVv6zaFIsR4YzhlJptZZSHv0M6ERqv9jPSGtp8po/edit?disco=AAABKNJuq68 which states we want to track the feature usage.

Reply:
After reading the comment, I'm a bit unsure on what should happen if the user removes all keywords. Is this still considered as using the feature? If not, then the number of keywords is needed to check if value is 0. Another alternative is to not fire this event if number of keywords is zero. 

@catalintomai any thoughts on that?

Reply:
> An alternative option is to not fire this event if number of keywords is zero.

Agreed added that check as well

Reply:
Agree, if the number of keywords is 0, there is no "using the feature" anymore (even though the user configured the feature using its UI).

Reply:
this is an interesting question: should we track people trying and disabling a feature? it might be a strong indicator that they didn't like/understand/can't apply the feature

Reply:
i will leave that upto @ifoukarakis @catalintomai to answer

Reply:
@M-ZubairAhmed , re: "number of keywords added, removed and total" my comment in https://docs.google.com/document/d/1u7UcVv6zaFIsR4YzhlJptZZSHv0M6ERqv9jPSGtp8po/edit?disco=AAABKNJuq68  should not be interpreted as excluding properties. Properties are useful to provide more context and help discriminate between different users/customers (e.g. customer X in industry Y  is more likely to have users that use a large number of keywords, pertaining to their specific use cases). 
re: "should we track people trying and disabling a feature" - the current definition is - if the feature is disabled (number of keywords == 0), it is not "in usage". What would be also interesting, indeed, is - has the user used the feature and turned it off later - basically measuring feature churn. This is not part of the requirements for the current work, but might be something to look for in the future.